PolarisO365.ps1

```diff
@@ -767,6 +767,229 @@ function Get-PolarisO365OneDrive() {
     return $user_details
 }
 
+function Get-PolarisO365SharePoint() {

```

Comment:
In the QUICK_START.md, could you add this newly added function in the quick start instruction?